src/_scss/pages/homepage/features/_covidFeature.scss

```
@@ -0,0 +1,238 @@
+                    display: block;
+
+                    max-width: 75%;
+                    border: rem(1) solid #D6D7D9;
+                    padding: rem(10);
+                    margin-top: rem(0);
+                    margin-bottom: 5%;
+                    background-color: #FFFFFF;
+                }
+            }
+        }
+
+        .adv-search-spending-profile__text {
+            @include media ($medium-screen) {
+                max-width: 85%;
+            }
+        }
+
+        .feature-covid__button-wrap {
+            margin-top: auto;
+            padding-top: 3.5rem;
+            width: 19rem;
+            height: 5.7rem;
+            margin-left: 0;
+
+            .feature-covid__button {
+                display: block;
+
+                color: $color-white;
+                background-color: $color-primary;
+                padding: rem(15) rem(20);
+
+                font-size: rem(21);
+
+                &:visited {
+                    color: $color-white;                    
+                }
+
+                &:hover, &:active {
+                    text-decoration: none;
+                    background-color: $color-primary-darker;
+                }
+            }
+        }
+    }
+    //ROW 3: AWARD SUMMARY 
+    .award-summary__wrapper {
+        margin-top: rem(60);
+
+        @include media ($medium-screen) {
+            @include display(flex);
+            @include justify-content(center);
+            padding-left: rem(70);
+            padding-right: rem(70);
+        }
+
+        .feature-covid-award-summary__image-wrapper {
+            @include display(flex);
+            @include justify-content(flex-start);
+
+            .feature-covid-award-summary__image-mobile {
+                display: block;
+                border: rem(1) solid #D6D7D9;
+                padding: rem(10);
+                margin-bottom: 5%;
+                background-color: #FFFFFF;
+                width: 85%;
+                
+                margin-top: rem(20);
+                margin-left: auto;
+                margin-right: auto;
+
+                z-index: 1;
+
+                @include media($medium-screen) {
+                    display: none;
+                }
+
+            }
+
+            .feature-covid-award-summary__image {
+                display: none;
+
+                @include media($medium-screen) {
+                    display: block;
+                    border: rem(1) solid #D6D7D9;
+                    padding: rem(10);
+                    margin-bottom: 5%;
+                    background-color: #FFFFFF;
+                    width: 100%;
+                    z-index: 1;
+                }
+            }
+
+        }
+    
+        .award-summary__text-wrapper {
+            margin-left: 0;
+            z-index: 1;
+            position: relative;
+
+            @include media($medium-screen) {
+                margin-left: rem(55);
+                max-width: 45%;
```

Max Width is not gonna work in IE. Lets must make this width

```suggestion
                width: 45%;
```